spec/mandrill_dm/message_spec.rb

```diff
@@ -131,6 +131,16 @@ def new_mail(options = {}, &blk)
       expect(message.global_merge_vars).to eq(global_merge_vars)
     end
 
+    it 'does not use gsub to convert string into JSON' do
```

Comment:
ok, what I'm trying to say is that gsub/json.parse may not be the only way to incorrectly restore/reanimate the hash (fancy regex then parse, or small lexer/parser, etc).  So, what I'm after is a more general description for the test that doesn't mention the *implementation* (in general tests aren't supposed to know anything about the implementation).  Perhaps `retrieves/restores the hash to original state` captures the idea?  

I already added a `looks good to me`, so that means you are free to merge when you are ready.  These test descriptions are just small semantic points.